Contents/Code/__init__.py

```diff
@@ -191,27 +191,47 @@ def search(self, results, media, lang, manual):
 
         info = self.run_search(search_helper, result)
         
-        # Set localized "by"
-        by_lang_dict = {
-            Locale.Language.English: 'by',
-            'de': 'von',
-            'fr': 'de',
-            'it': 'di'
+        # Nested dict for localized separators
+        # 'T_A' is the separator between title and author
+        # 'A_N' is the separator between author and narrator
+        sep_dict = {
+            Locale.Language.English: {'T_A': 'by', 'A_N': 'with'},
+            'de': {'T_A': 'von', 'A_N': 'mit'},
+            'fr': {'T_A': 'de', 'A_N': 'avec'},
+            'it': {'T_A': 'di', 'A_N': 'con'}
         }
-        localized_sep = by_lang_dict.get(lang, "by")
-        log.debug("Localized separator between title and artist: " + localized_sep)
+        local_seps = sep_dict[lang]
+        log.debug(
+            'Using localized separators "%s" and "%s"',
+            local_seps['T_A'], local_seps['A_N'] 
+        )
 
         # Output the final results.
         log.separator(log_level="debug")
         log.debug('Final result:')
         for i, r in enumerate(info):
-            description = '\"%s\" %s %s' % (
-                r['title'], localized_sep, r['artist']
+            # Truncate long titles
+            # Displayable chars is ~60 (see issue #32)
+            # Inlcude tolerance to only truncate if >4 chars need to be cut
+            title_trunc = (r['title'][:32] + '..') if len(
+                r['title']) > 38 else r['title']
+            
+            # Shorten artist
+            artist_inits = self.name_to_initials(r['artist'])
+            # Shorten narrator
+            nr_inits = self.name_to_initials(r['narrator'])
```

Comment:
It's always better to use clear variable names than to use short names. Also, because object constructors use `__init__`, I think it's best to just use the full word for initials. So:
`artist_initials`
`narrator_initials`

This will make what these variables do very clear for future contributors.